### PR TITLE
Iframe tag support for DCM web device mode

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Updated web device mode settings for DCM Floodlight with the new iframe tag support.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Comment by Gauravudia** [comment]
@ameypv-rudder Can we also add Iframe mapping mentioned [here](https://www.notion.so/rudderstacks/DCM-Floodlight-Web-Device-Mode-using-iframe-tag-415c3214cacb4808bd00e62ed1b5f716#71c2ca271d404c05b67e7984bf2a22ea) in the DOC.

File: docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx [hunk 1]
```diff
@@ -47,6 +47,24 @@ The following table details the mapping between RudderStack and DCM Floodlight's
...
 
+There are some slight variations in terms of the mapped DCM Floodlight parameters when you use the Iframe tag, as opposed to the global site tag. However, RudderStack handles this variation internally.
+
+The following table details the mapping between the optional RudderStack and DCM Floodlight properties when using the Iframe tag:
+
+| RudderStack property | DCM Floodlight property | Tag |
+| :--- | :--- |:--- |
+| `properties.ord` | `ord` | Counter - standard |
+| `properties.sessionId` | `ord` | Counter - per_session  |
+| `properties.sessionId` | `ord=1` | Counter - unique |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx:58]: `properties.sessionId` keep this as blank . We are sending ord=1 when activity is Counter -unique

File: docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx [hunk 2]
```diff
@@ -47,6 +47,24 @@ The following table details the mapping between RudderStack and DCM Floodlight's
...
+| `properties.num` | `num` | Counter - unique |
+| `context.device.adTrackingEnabled` | `dc_lat` | Counter/Sales |
+| `properties.matchId` | `match_id` |  Counter/Sales |
+| `properties.orderId` | `ord` |  Sales |
+| `properties.orderId` | `transaction_id` |  Sales |
+| `properties.revenue` | `cost` |  Sales |
+| `properties.quantity` | `qty` | Sales - items_sold |
+| `properties.quantity` | `qty=1` | Sales - transactions |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx:66]: `properties.quantity `keep this as blank. We are sending qty=1 when activity is Sales - transactions

**Review by Gauravudia — CHANGES_REQUESTED** [review]

File: docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx:66]: Screen Shot 2022-10-31 at 7 53 23 PM

Does this format of presentation work better?

Comment (Gauravudia) [reply docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx:66]: `properties.sessionId` is not applicable for Counter -unique.

Comment (Gauravudia) [reply docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx:66]: `sessionId` is related to session :- `Counter- per_session`

Comment (Gauravudia) [reply docs/destinations/streaming-destinations/dcm-floodlight/dcm-floodlight-device-mode.mdx:66]: Updated the mapping [here](https://www.notion.so/rudderstacks/DCM-Floodlight-Web-Device-Mode-using-iframe-tag-415c3214cacb4808bd00e62ed1b5f716#71c2ca271d404c05b67e7984bf2a22ea) please check.

**Review by Gauravudia — APPROVED** [review]

**Pull request merged**